.github/workflows/ci.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
       - uses: actions/upload-artifact@v4
         if: success() || failure()
         with:
-          name: ftw-envoy-logs
+          name: ftw-envoy-logs-multiphase-${{ matrix.multiphase_eval }}
```

Comment:
To avoid `Error: Failed to CreateArtifact: Received non-retryable error: Failed request: (409) Conflict: an artifact with this name already exists on the workflow run`